Source/SwiftLintFramework/Rules/RuleConfigurations/InclusiveLanguageConfiguration.swift

```diff
@@ -0,0 +1,49 @@
+private enum ConfigurationKey: String {
+    case severity
+    case allow
+    case deny
+}
+
+public struct InclusiveLanguageConfiguration: RuleConfiguration, Equatable {
+    public var severityConfiguration = SeverityConfiguration(.warning)
+    public var denyList: Set<String>
+
+    public var consoleDescription: String {
+        severityConfiguration.consoleDescription
+            + ", deny: \(denyList.sorted())"
```

Comment:
i think it's worth keeping the values from configuration in memory so we can print here the exact configuration from `.swiftlint.yml` instead of the "resolved" one